src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -202,22 +206,28 @@ export const SecurityAnalysisResultTab: FunctionComponent<
     const result = useMemo(
         () =>
             securityAnalysisResult === RunningStatus.FAILED
-                ? null
+                ? []
                 : securityAnalysisResult,
         [securityAnalysisResult]
     );
 
-    const [filterEnumsLoading, filterEnums] = useFetchFiltersEnums(
-        result?.empty
-    );
+    const { loading: filterEnumsLoading, result: filterEnums } =
+        useFetchFiltersEnums(hasFilter, setHasFilter);
 
     useEffect(() => {
         if (result) {
-            setCount(result.totalElements);
+            switch (tabIndex) {
+                case 0:
+                    setCount(result.length);
+                    break;
+                case 1:
+                    setCount(result.totalElements);
+                    break;
+            }
```

Comment:
why switch ?
maybe:
```suggestion
           setCount(tabIndex? result.totalElements:result.length)
```

Reply:
Done